src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.html

```diff
@@ -0,0 +1,16 @@
+<ng-container *ngIf="org; else noOrg">
+  <h1 class="govuk-heading-xl">Confirm your decision</h1>
+  <app-pba-account-decision *ngFor="let pba of org.pbaNumber; let i = index" [pbaNumber]="pba"
+    [accountName]="org.accountDetails[i]?.account_name" [organisationId]="org.organisationId"
+    [decision]="newPBAs.get(pba) === 'approve' ? 'Approve the account' : 'Reject the account'">
```

Comment:
maybe could move this logic into the component, makes it testable etc

Reply:
Fair comment. It's currently getting this out of the ngrx store and I didn't want to refactor it at this stage as moving forward (one of the next few tickets) we are going to be making calls to the node layer for this information, at which point the logic will be in the component and will be readily testable.